### PR TITLE
Chore: use `ScrollContainer` across frontend platform components

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- replaces use of `CustomScrollbar` with `ScrollContainer` in frontend platform components
- follow up to https://github.com/grafana/grafana/pull/94689

windows screenshots:
| | before | after |
| --- | --- | --- |
| `DataLinkInput` | image | image |
| `RelativeTimeRangePicker` | image | image |
| `Drawer` | image | image |
| `Select` | image | image |
| `TabbedContainer` | image | image |
| `MegaMenu` | image | image |

**Why do we need this feature?**

- consistent scrollbars on each platform
- better a11y

**Who is this feature for?**

- everyone!

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx [hunk 1]
```diff
@@ -157,14 +157,16 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps) {
...
-                  <CustomScrollbar className={styles.leftSide} hideHorizontalTrack>
-                    <TimeRangeList
-                      title={t('time-picker.time-range.example-title', 'Example time ranges')}
-                      options={validOptions}
-                      onChange={onChangeTimeOption}
-                      value={timeOption}
-                    />
-                  </CustomScrollbar>
+                  <div className={styles.leftSide}>
+                    <ScrollContainer showScrollIndicators>
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:161]: Any reason we don't have `className` prop on `ScrollContainer?

Comment (ashharrison90) [reply packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:161]: same reason we don't have it on any other of our components really. we don't want handcrafted css interfering with our component styles. it's still in `unstable` so if we find we actually need it we can add it

Comment (Clarity-89) [reply packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:161]: I thought we've discussed somewhere that it's ok to expose `className`, but maybe I misremember 🤔

**Review by Clarity-89 — APPROVED** [review]
🚀

**Pull request merged**